static/js/tools/map/compute_hook.ts

```diff
@@ -13,18 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+// This module contains custom React hooks that makes computation for map chart.
+
 import _ from "lodash";
-import { Dispatch, useContext, useEffect } from "react";
+import { Dispatch, useContext, useEffect, useMemo } from "react";
 
 import { GeoJsonData } from "../../chart/types";
 import { ChartDataType, ChartStore, ChartStoreAction } from "./chart_store";
 import { Context } from "./context";
 import { useDefaultStatReady, useGeoJsonReady } from "./ready_hook";
 import { getGeoJsonDataFeatures, MANUAL_GEOJSON_DISTANCES } from "./util";
 
-export function useCalculateRatio() {
+// Custom hook to check if needs to compute ratio for the stat.
+export function useIfRatio() {
```

Comment:
I'm wondering if this hook belongs in this file because from the high level comment about this file, sounds like this file will contain hooks that do computations? and this hook is not doing any kind of computation

Reply:
moved this to condition_hooks.ts